README.md

```diff
@@ -150,6 +150,10 @@ Run `docker-compose up -d` instead.
 
 * Permission problem? See [this doc (Setting up Permission)](http://symfony.com/doc/current/book/installation.html#checking-symfony-application-configuration-and-setup)
 
+* How to config Xdebug?
+Xdebug is configured out of the box!
+Just config your IDE to connect port  `9001` and id key `PHPSTORM`
```

Comment:
This needs more clarification.
Whenever I'm trying to enable PHPStorm's listening function, I get the error "Can't start listening for connections from 'xdebug': Port 9000 is busy", no matter what I try. Perhaps you would be so kind as to elaborate on what you mean by this?